### PR TITLE
Added Infrastructure of tile :D

### PR DESCRIPTION
**Pull request by Beta-King**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
[If any]

### Links

http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by abishekvashok** [comment]
Can you add their implementation as well.

**Comment by Beta-King** [comment]
Yup thats what another GitHub user is doing -- we are collaborating :D

**Comment by abishekvashok** [comment]
Pls link that here once done.

**Comment by Beta-King** [comment]
Link what ??

**Comment by abishekvashok** [comment]
> Link what ??

Link that Pr.

**Comment by Beta-King** [comment]
Ohkay Yeah :D

**Comment by Beta-King** [comment]
@abishekvashok --> done changes :D

**Comment by Beta-King** [comment]
Thanks :D @samyakbvs

**Comment by Beta-King** [comment]
@niccokunzmann --> can we merge ??

**Comment by Beta-King** [comment]
Thanks :D @arjunpandey2003

File: tiles/rooms/wall/redWall.svg [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+     id="path14"
+     inkscape:connector-curvature="0" /><path
+     style="fill:#ff2a2a;stroke-width:1.42857146"
+     d="m 430.13298,162.74149 c 0,-19.25 0.20221,-27.125 0.44937,-17.5 0.24715,9.625 0.24715,25.375 0,35 -0.24716,9.625 -0.44937,1.75 -0.44937,-17.5 z"
+     id="path16"
+     inkscape:connector-curvature="0" /><path
+     style="fill:#ff2a2a;stroke-width:1.42857146"
+     d="m 422.37234,182.24944 c 0,-9.6992 -0.41514,-24.41994 -0.92254,-32.71276 l -0.92255,-15.07785 4.49398,-2.32393 4.49397,-2.32392 v 33.09348 c 0,29.84226 -0.24561,33.27993 -2.5,34.99125 -1.375,1.04378 -2.98214,1.91822 -3.57143,1.94321 -0.58928,0.025 -1.07143,-7.89028 -1.07143,-17.58948 z"
+     id="path18"
+     inkscape:connector-curvature="0" /></svg>
```
Comment (sayak119) [inline tiles/rooms/wall/redWall.svg]: Add a new line at the end.

File: tiles/rooms/door/redDoor.svg [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+       style="fill:#ff2a2a;stroke-width:0.50507629"
+       d="M 5.0859168,131.27553 C 2.6682915,129.84169 0.73637473,128.6283 0.79276867,128.57911 0.84916256,128.52991 49.148288,99.708134 108.12416,64.530713 l 107.22885,-63.95895371 4.56622,2.73743361 4.56622,2.7374338 -11.03236,6.5582453 C 207.3853,16.211907 159.15446,44.986823 106.27345,76.549129 53.392435,108.11144 9.9811312,133.9233 9.8038782,133.90882 9.6266247,133.89433 7.5035427,132.70936 5.0859168,131.27553 Z"
+       id="path25"
+       inkscape:connector-curvature="0"
+       transform="translate(53.643411,569.625)" /><path
+       style="fill:#ff2a2a;stroke-width:0.50507629"
+       d="m 4.8096441,199.31313 -4.67195546,-2.80345 v -33.81265 c 0,-18.59696 0.1301821,-33.81265 0.28929349,-33.81265 0.1591114,0 2.31288897,1.19324 4.78617247,2.65165 l 4.4968789,2.65165 0.012051,33.96638 c 0.00663,18.68151 -0.04477,33.96551 -0.1142169,33.96445 -0.069448,-10e-4 -2.2286491,-1.26348 -4.7982246,-2.80538 z"
+       id="path27"
+       inkscape:connector-curvature="0"
+       transform="translate(53.643411,569.625)" /></g></svg>
```
Comment (sayak119) [inline tiles/rooms/door/redDoor.svg]: Add a new line at the end

**Comment by Beta-King** [comment]
@niccokunzmann --> there is something wrong with Travis, can we merge ??

**Comment by niccokunzmann** [comment]
Thanks, yes, you can fix the build if you like.

**Comment by Beta-King** [comment]
Thanks, yeah ill try :D

**Pull request merged**